crates/iota-graphql-client/src/query_types/coin.rs

```diff
@@ -34,16 +34,22 @@ use crate::query_types::{BigInt, schema};
 #[cynic(schema = "rpc", graphql_type = "CoinMetadata")]
 pub struct CoinMetadata {
     /// The number of decimal places used to represent the token.
+    #[cfg_attr(feature = "uniffi", uniffi(default = None))]
```

Comment:
Is this really needed for each field individually?

Reply:
If uniffi doesn't provide something, is there not a good way to write an own macro that adds it for Option<> fields?

Reply:
Had a look, but seems to be rather complicated, probably not worth it